src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -80,6 +80,16 @@ public function followRedirects($followRedirect = true)
     }
 
     /**
+     * Returns whether client automatically follow redirects or not.
+     *
+     * @return bool
+     */
+    public function getFollowRedirects()
```

Comment:
As this a boolean value it should be something like `is[...]()`.
`isRedirectFollowed()`